### PR TITLE
Release/10

### PR DESCRIPTION
**Pull request by StanislasGuinel**

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+        "description": "Plugin for optical character recognition (OCR) in python using the tesseract engine.",
+
+        "author": "Dataiku (Stanislas GUINEL)",
+
+        "icon": "icon-folder-close",
+
+        "tags": ["NLP", "Images"],
+
+        "url": "https://www.dataiku.com/product/plugins/tesseract-ocr.html",
```
Comment (MayeulRousselet) [inline plugin.json:18]: The url page is not well formatted, use https://www.dataiku.com/product/plugins/tesseract-ocr/ instead

File: custom-recipes/image-conversion/recipe.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        pdf_images = convert_from_bytes(img_bytes, fmt='jpg')
+
+        for i, img in enumerate(pdf_images):
+            img = img.convert('L')
+            buf = BytesIO()
+            img.save(buf, format='JPEG')
+            img_bytes = buf.getvalue()
+            output_folder.upload_data("{}_pdf_page_{:05d}.jpg".format(prefix, i+1), img_bytes)
+
+    elif suffix in ["jpg", "png", "jpeg", "tiff"]:
```
[the remark this answers is not in the document]
Comment (StanislasGuinel) [reply custom-recipes/image-conversion/recipe.py:31]: It is converting the image to greyscale jpg. I factored the code for better understanding.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,73 @@
...
+
+Tesseract must be installed on DSS instance server:
+
+#### Mac
+Using homebrew: `brew install tesseract`
+Using macports: `sudo port install tesseract`
+
+#### Linux
+Tesseract is available directly from many Linux distributions. 
+`sudo apt install tesseract-ocr`
```
[the remark this answers is not in the document]
Comment (StanislasGuinel) [reply README.md:43]: I added yum as well, but for more complex set up, I provide a link to the installation doc.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,73 @@
...
+Mac users will have to install poppler for Mac (<http://macappstore.org/poppler/>).
+
+#### Linux
+Most distros ship with pdftoppm and pdftocairo. If they are not installed, refer to your package manager to install poppler-utils
+
+For more informations, go to: <https://github.com/Belval/pdf2image>.
+
+### Specific languages
+
+If you want to specify languages in tesseract, you must install them on your DSS instance, you can find instructions on how to install them and the code for each language here <https://tesseract-ocr.github.io/tessdoc/Data-Files>.
```
[the remark this answers is not in the document]
Comment (StanislasGuinel) [reply README.md:62]: So i didn't find a simple way to install a specific language with brew, instead users must install all languages with `brew install tesseract-lang`.

**Pull request merged**